packages/dashboard-core-plugins/src/linker/Linker.tsx

```diff
@@ -108,9 +109,15 @@ export class Linker extends Component<LinkerProps, LinkerState> {
     this.handleStateChange = this.handleStateChange.bind(this);
     this.handleExited = this.handleExited.bind(this);
     this.handleLinkSelected = this.handleLinkSelected.bind(this);
+    this.handlePanelDragging = this.handlePanelDragging.bind(this);
+    this.handlePanelDropped = this.handlePanelDropped.bind(this);
     this.isColumnSelectionValid = this.isColumnSelectionValid.bind(this);
 
-    this.state = { linkInProgress: undefined, selectedIds: new Set<string>() };
+    this.state = {
+      linkInProgress: undefined,
+      selectedIds: new Set<string>(),
+      isDraggingPanel: true,
```

Comment:
I think you've got the `boolean` backwards for the way this is named. `isDraggingPanel` should be `true` while there's a panel that's currently being dragged.